packages/kbn-i18n/package.json

```diff
@@ -5,10 +5,5 @@
   "types": "./target/types/index.d.ts",
   "version": "1.0.0",
   "license": "SSPL-1.0 OR Elastic License 2.0",
-  "private": true,
-  "scripts": {
-    "build": "node scripts/build",
-    "kbn:bootstrap": "node scripts/build --source-maps",
-    "kbn:watch": "node scripts/build --watch --source-maps"
-  }
+  "private": true
```

Comment:
Do we still need the `scripts/build` source?

Reply:
we don't, i forgot to commit those! Thanks for catching it 😃